packages/mobile/src/screens/home/components/product.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FC } from 'react';
+import { Text, View, Button, Image } from '~/components/components';
+import { ButtonType, ButtonAppearance } from '~/common/enums/enums';
+import { images } from '~/assets/pictures/images/images';
+import { styles } from './style';
+
+type Props = {
+  title: string;
+  description: string;
+  price: number;
+  imageUrl?: string;
+};
+
+export const Product: FC<Props> = ({ title, description, price, imageUrl }) => {
```

Comment:
I'd suggest extracting image of stamp and dedicating this component for one specific purpose -- displaying the lot card.
Also if you making just mocked card there is no sense in passing props, you can place text directly here.